packages/backend/src/graphql/schema.graphql

```diff
@@ -241,6 +241,14 @@ type Mutation {
     input: CreateWalletAddressKeyInput!
   ): CreateWalletAddressKeyMutationResponse
 
+  createTenantSettings(
+    input: CreateTenantSettingsInput!
+  ): CreateTenantSettingsMutationResponse
+
+  deleteTenantSettings(
```

Comment:
this makes me think, do we want to have this mutation in general? since if a tenant gets rid of their `EXCHANGE_RATES_URL` for example, it would be bad 😅

Reply:
I think that maybe we do not need it. I don't think that we have some string based config that can be optional for example. Should I remove this mutation?

Reply:
Yes, I think we can remove it for now